src/applications/vaos/tests/e2e/workflows/appointment-list-workflow/upcoming-appointments.cypress.spec.js

```diff
@@ -976,56 +988,15 @@ describe('VAOS upcoming appointment flow', () => {
       cy.axeCheckBestPractice();
     });
 
-    it('should display appointment details for VA video appointment', () => {
```

Comment:
Is this not a valid scenario anymore?

Reply:
@vbahinwillit , this scenario is an exact test to "should display appointment details for clinic video appointment" on line 847. I just removed the duplicate.